docs/src/design/stack/io_ops.md

```diff
@@ -68,16 +68,14 @@ $$
 Using the above variable, we define the value representing the memory access request as follows:
 
 $$
-u_{mem} = \alpha_0 + \alpha_1 \cdot op_{mem\_read} + \alpha_3 \cdot s_0 + \alpha_4 \cdot clk + v
+u_{mem} = \alpha_0 + \alpha_1 \cdot op_{mem\_read} + \alpha_2 \cdot ctx + \alpha_3 \cdot s_0 + \alpha_4 \cdot clk + v
```

Comment:
I would add a bullet point for $ctx$ below (i.e., on line 77) with a brief explanation of what it is, and would also make similar updates to other operations in this file.